tracing-attributes/src/lib.rs

```diff
@@ -160,7 +155,16 @@ pub fn instrument(args: TokenStream, item: TokenStream) -> TokenStream {
     // enough Rust version to support this. Otherwise, this will enter the span
     // and then perform the rest of the body.
     let body = if asyncness.is_some() {
-        async_await::gen_async_body(block, &call_site)
+        let span = block.span();
+        let async_kwd = syn::token::Async { span };
```

Comment:
I'd explain in comment above why you're doing this for a reader that might not have this PR on hand.

Reply:
good call, thanks